### PR TITLE
fix: add labels to service examples

### PR DESCRIPTION
**Pull request by Andras-Csanyi**
## PR summary

This PR adds the following:

- labels to examples, so it is possible to identify which output section belongs to which method when examples are run
- it also replaces "\n" to "%n" in printf methods. "%n" is platform indepented new-line character

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by padamstx — APPROVED** [review]
LGTM

File: modules/examples/src/main/java/com/ibm/cloud/platform_services/open_service_broker/v1/OpenServiceBrokerExamples.java [hunk 1]
```diff
@@ -81,29 +81,6 @@ protected OpenServiceBrokerExamples() { }
...
-// OPEN_SERVICE_BROKER_BINDING_ID=<The ID of a previously provisioned binding for that service instance>
-// OPEN_SERVICE_BROKER_SPACE_GUID=<The identifier for the project space within the IBM Cloud platform organization>
-// OPEN_SERVICE_BROKER_APPLICATION_GUID=<GUID of an application associated with the binding>
-// OPEN_SERVICE_BROKER_ORGANIZATION_GUID=<The IBM Cloud platform GUID for the organization under which the service instance is to be provisioned>
-//
-// These configuration properties can be exported as environment variables, or stored
-// in a "credentials" file and then:
-// export IBM_CREDENTIALS_FILE=<name of credentials file>
-//
-
```
Comment (pyrooka) [inline modules/examples/src/main/java/com/ibm/cloud/platform_services/open_service_broker/v1/OpenServiceBrokerExamples.java:106]: Why this has been removed?

**Review by pyrooka — APPROVED** [review]
Looks good!

File: modules/examples/src/main/java/com/ibm/cloud/platform_services/open_service_broker/v1/OpenServiceBrokerExamples.java (hunk 1 above)
Comment (Andras-Csanyi) [reply modules/examples/src/main/java/com/ibm/cloud/platform_services/open_service_broker/v1/OpenServiceBrokerExamples.java:106]: It was a duplicated section within the file.
The information still available in the file, you can check it here:
https://github.com/Andras-Csanyi/platform-services-java-sdk/blob/f07da2eca6c0622ef07e623fdbf331a36377e6f0/modules/examples/src/main/java/com/ibm/cloud/platform_services/open_service_broker/v1/OpenServiceBrokerExamples.java#L42

Comment (pyrooka) [reply modules/examples/src/main/java/com/ibm/cloud/platform_services/open_service_broker/v1/OpenServiceBrokerExamples.java:106]: I see, thanks!

File: modules/examples/src/main/java/com/ibm/cloud/platform_services/case_management/v1/CaseManagementExamples.java [hunk 2]
```diff
@@ -105,7 +105,7 @@ public static void main(String[] args) throws Exception {
       Response<Case> response = service.createCase(createCaseOptions).execute();
       Case xCase = response.getResult();
 
-      System.out.println(xCase);
+      System.out.printf("createCase() response: %n %s %n", xCase.toString());
```
Comment (pyrooka) [inline modules/examples/src/main/java/com/ibm/cloud/platform_services/case_management/v1/CaseManagementExamples.java:108]: Same thing about the indentation that I've mentioned in the Go and Python repositorties:
```
uploadFile() response: 
 {
  "id": "xxx",
  "filename": "example.log",
  "size_in_bytes": 42,
  "created_at": "2021-03-23 17:04:18",
  "url": "xxx"
} 
```
```suggestion
      System.out.printf("createCase() response: %n%s%n", xCase.toString());
```

**Review by pyrooka — CHANGES_REQUESTED** [review]
I request this changes just because to make sure this PR won't be merged before we discuss this.

File: modules/examples/src/main/java/com/ibm/cloud/platform_services/case_management/v1/CaseManagementExamples.java (hunk 2 above)
Comment (padamstx) [reply modules/examples/src/main/java/com/ibm/cloud/platform_services/case_management/v1/CaseManagementExamples.java:108]: I agree.  In your Go PR I mentioned the need to be consistent with the wording across all 4 SDKs.  So let's use "result" in the label, like this:
```
System.out.printf("createCase() result: %n%s%n", xCase.toString());
```

**Review by padamstx — CHANGES_REQUESTED** [review]
Upon further review, let's make sure that the wording that we'll use within the labels is consistent across all 4 PRs.

**Review by padamstx — APPROVED** [review]
LGTM

**Review by pyrooka — APPROVED** [review]
👍

**Pull request merged**